VideoLocker/src/main/java/org/edx/mobile/module/notification/ParseSyncTask.java

```diff
@@ -0,0 +1,36 @@
+package org.edx.mobile.module.notification;
+
+import android.content.Context;
+
+import com.parse.ParseInstallation;
+
+import org.edx.mobile.model.api.AuthResponse;
+import org.edx.mobile.task.Task;
+
+import java.util.List;
+
+/**
+ * Sync with Parse server
+ */
+public abstract class ParseSyncTask extends Task<Void> {
```

Comment:
@aleffert  using progress bar is an option for Task.  I prefer to use Task as it is our common super class for all AsyncTask.